packages/aragon-wrapper/src/rpc/handlers/sign-message.js

```diff
@@ -0,0 +1,4 @@
+export default function (request, proxy, wrapper) {
+  const messageToSign = request.params[0]
+  return wrapper.signMessage({ fromAddress: proxy.address, message: messageToSign })
```

Comment:
wondering if this is a bit confusing. I at first thought `fromAddress` should be the address that should sign the message, not the app that sent the request.

Should we include both addresses? The front-end already has access to the `account` that will sign the message, so the wrapper doesn't _need_ to send this information along the way. Although, on the other hand, the `account` should never really differ?